src/command/revoke.iced

```diff
@@ -88,9 +86,13 @@ exports.Command = class Command extends Base
   run : (cb) ->
     esc = make_esc cb, "run"
     await session.login esc defer()
+    await session.get_email_or_username_i esc defer email_or_username
```

Comment:
No, it gets it from the session. I think the call to `login` above guarantees that it'll be there.